src/pages/AccountPage/ProfilePage.jsx

```diff
@@ -124,15 +127,46 @@ const ProfilePage = ({ user = {}, isLoading }) => {
     }
   }
 
+
+  const onUpdateAvatar = async (file) => {
+    try {
+      const user_name = user.name
+      const imageKey = `?${Math.random().toString(36).substring(2, 15)}-${Date.now()}`
+      const opts = {
+        headers: {
+          'Content-Type': file.type,
+        },
+      }
+      await axios.put(`/api/users/${user_name}/avatar`, file, opts)
+      toast.success('Profile updated')
+      // update redux state with new data
+      dispatch(updateUserAttribs(formData))
+      // reset form
+      setInitData(formData)
+      setImageKey(imageKey)
+      setChangesMade(false)
```

Comment:
Updating the avatar and saving the form are two completely different actions. The form state should not be affected. Remove lines 142 to 147

Reply:
Done.